amr-wind/wind_energy/ABLWallFunction.cpp

```diff
@@ -349,11 +379,17 @@ void ABLVelWallFunc::operator()(Field& velocity, const FieldState rho_state)
 
 ABLTempWallFunc::ABLTempWallFunc(Field&, const ABLWallFunction& wall_fuc)
     : m_wall_func(wall_fuc)
-{}
+{
+    amrex::ParmParse pp("HeatFlux");
```

Comment:
Can reuse `ABL.wall_shear_stress_type` here? I don't think that we have a use-case where we would do things differently for vel/temperature, is there? So one input parameter would suffice?